### DISCUSSION TITLE
How do you feel about Discussions?

### DISCUSSION THREAD

**Discussion by jackmcdade** [statamic/cms#4476] [General]
Does anyone miss [the forum](https://statamic.com/forum)? Or is this a positive step forward?

**Comment by fitzage** [comment]
Ummm…wow…I wasn't paying attention. I didn't even know Discussions was a thing.

**Comment by eliasu** [comment]
Positive Step forward. I like, how Discussions works. Very responsive, good overview, clean styles, easy to use.

**Comment by niloc7** [comment]
I miss the bespokeness of the old forums :P It was really unique.. 

But yeah I guess to get down to business, github seriousness/profesionalism is a right step..

**Comment by robdekort** [comment]
I think the UI is a very messy and bloated but it makes a lot of sense to move it all here! Good call.

**Comment by zunnur-tiara** [comment]
I wish we can mark answered for thread reply.

**Reply by jackmcdade** [reply]
You can if you make it a question type discussion.

**Reply by zunnur-tiara** [reply]
Ahhh I didn't know that. But hopefully they implement for all too because too many unanswered discussion that actually answered.

**Comment by jensolafkoch** [comment]
Any chance to move over the v3 threads or threads started after a given date on the old forum? Single source of wisdom would be great. :-)

**Reply by jackmcdade** [reply]
Alas no I don’t foresee that happening, I don’t think it’s possible, and we wouldn’t be able to link up usernames and accounts and all that. 

The forums are still up, you can search anytime.

**Reply by kylef-fotf** [reply]
Is this still true? It looks like forums are now inaccessible, which is a bummer because nearly all search engine results point there. I know it's all about a deprecated version, but a lot of solutions still apply or at least loosely apply. I understand it can't be up forever.

**Reply by jackmcdade** [reply]
The forum posts were really more confusing than helpful at this point, as most were for v2. It was time, and yes they're all permanently gone now. The search results should recalibrate sooner or later to other content that's actually up to date.

**Reply by earthwytch** [reply]
@kylef-fotf I've been using Google's cached version of the old forum results for when I get desperate and can't get an answer here or find one online. It's been helpful to point me vaguely in the right direction in a few situations.

![image](https://github.com/statamic/cms/assets/4753015/3fc4da35-4725-439c-a27f-5227d4beb8b3)
